File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        Map<Integer,Integer> m=new HashMap<>();
        for(int n:nums){
            m.put(n,m.getOrDefault(n,0)+1);
        }
        int max=0;
        for(int v:m.values()){
            if(v>max){
                max=v;
            }
        }
        int c=0;
        for(int val: m.values()){
            if(val==max){
                c+=1;
            }
        }
        return c*max;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        d={}
        for i in nums:
            if i in d:
                d[i]+=1
            else:
                d[i]=1
        m=0
        for k,v in d.items():
            if v>m:
                m=v
        c=0
        for k,v in d.items():
            if v==m:
                c+=m
        return c
